common/network-common/src/test/java/org/apache/spark/network/TransportRequestHandlerSuite.java

```diff
@@ -39,7 +39,7 @@
 public class TransportRequestHandlerSuite {
 
   @Test
-  public void handleStreamRequest() {
+  public void handleStreamRequest() throws Exception {
```

Comment:
is this change necessary?

Reply:
Yes, because of the changes for `TransportRequestHandler.handler` here: https://github.com/apache/spark/pull/27665/files#diff-0e3429029d3f8d49e94ef11e4e3051a2R105